packages/alfa-dom/test/compare-document-position.spec.tsx

```diff
@@ -23,3 +24,17 @@ test("Returns < 0 when the first node comes before the second", t => {
 test("Returns > 0 when the first node comes after the second", t => {
   t(compareDocumentPosition(bar, foo, context) > 0);
 });
+
+test("Returns 35 or 37 if the nodes are not in the same tree", t => {
+  const div = <div />;
+  const cmp = compareDocumentPosition(bar, div, context);
+  t(cmp === 35 || cmp === 37);
+});
+
+test("Returns -20 when the other node is contained by the reference node", t => {
+  t.equal(compareDocumentPosition(foobar, foo, context), -20);
+});
+
+test("Returns 10 when the reference node is contained by the other node", t => {
+  t.equal(compareDocumentPosition(foo, foobar, context), 10);
+});
```

Comment:
Use the bit flags rather than the decimal values. The decimal values make no sense on their own and just seem like magic numbers.